<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicManiac</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>musicmaniac.ico</normaloff>
    <normalon>musicmaniac.ico</normalon>musicmaniac.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="tabSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="playlistView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTreeView" name="musicView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="settingsLayout">
      <item>
       <layout class="QHBoxLayout" name="ratingLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="ratingLabel">
          <property name="text">
           <string>Minimum rating : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="ratingSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="maxDurationLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="maxDurationLabel">
          <property name="text">
           <string>Max duration : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxDurationEdit">
          <property name="text">
           <string>100:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="minDurationLayout">
        <item>
         <widget class="QLabel" name="minDurationLabel">
          <property name="text">
           <string>Min duration : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minDurationEdit">
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="groupsLayout">
      <item>
       <widget class="QGroupBox" name="tagsGroupBox">
        <property name="title">
         <string>Tags</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="withoutKeywordsLayout">
           <item>
            <widget class="QLabel" name="wihtoutKeywordsLabel">
             <property name="text">
              <string>Without</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="withoutKeywordsView">
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="withoutButtonsLayout">
           <item>
            <widget class="QPushButton" name="outWithoutButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="inWithoutButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="availableKeywordsLayout">
           <item>
            <widget class="QLabel" name="availableKeywordsLabel">
             <property name="text">
              <string>Available</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="availableKeywordsView">
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="withButtonsLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="inWithButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="outWithButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="withKeywordsLayout">
           <item>
            <widget class="QLabel" name="withKeywordsLabel">
             <property name="text">
              <string>With</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="withKeywordsView">
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="artistsGroupBox">
        <property name="title">
         <string>Artists</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="availableArtistsLayout">
           <item>
            <widget class="QLabel" name="availableArtistsLabel">
             <property name="text">
              <string>Available</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="availableArtistsView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="artistButtonsLayout">
           <item>
            <widget class="QPushButton" name="inArtistButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="outArtistButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="selectedArtistsLayout">
           <item>
            <widget class="QLabel" name="selectedArtistsLabel">
             <property name="text">
              <string>Selected</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="selectedArtistsView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="genresGroupBox">
        <property name="title">
         <string>Genres</string>
        </property>
        <layout class="QHBoxLayout" name="genresLayout">
         <item>
          <layout class="QVBoxLayout" name="availableGenresLayout">
           <item>
            <widget class="QLabel" name="availableGenresLabel">
             <property name="text">
              <string>Available</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="availableGenresView">
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="genreButtonsLayout">
           <item>
            <widget class="QPushButton" name="inGenreButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="outGenreButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="selectedGenresLabel">
             <property name="text">
              <string>Selected</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="selectedGenresView">
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionOpenRegenFolder"/>
    <addaction name="actionRescanFolder"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionAdd_missing_MP3_UUID">
   <property name="text">
    <string>Add missing MP3 UUID</string>
   </property>
  </action>
  <action name="actionAdd_missing_FLAC_UUID">
   <property name="text">
    <string>Add missing FLAC UUID</string>
   </property>
  </action>
  <action name="actionOpenRegenFolder">
   <property name="text">
    <string>Open Folder (regen UUID)</string>
   </property>
  </action>
  <action name="actionAnd_support">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>And support</string>
   </property>
  </action>
  <action name="actionRefresh_Playlist">
   <property name="text">
    <string>Refresh Playlist</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate / Refresh</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRescanFolder">
   <property name="text">
    <string>Rescan Folder</string>
   </property>
  </action>
  <action name="actionGenerateBest">
   <property name="text">
    <string>Auto Generate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
